### DISCUSSION TITLE
Unable to Update V400 Motherboard Firmware (Nano V2.1 Clone)

### DISCUSSION THREAD

**Discussion by REPH26** [Guilouz/Klipper-Flsun-Speeder-Pad#66] [Q&A] [answered]
I formated the SD card in fat32 allocation 4096 copy the file Robin_nano35.bin to root
Turn off printer insert SD in motherboard SD slot turn on white led on motherboard and nothing….
Take off the SD card the file is not renamed 
Can you help me to update this firmware?
On the procedure I just answered yes for adding recommanded macro
And I tried firmware update of BigTreeTech SKR 3.0 before I understand I’m on Nano v2.1
Thank you for your help

**Comment by JulianGroeli** [comment, marked as answer]
How big is the SD card? When I first tried this with a 64GB card it didn't work. Then I tried again with a 16GB card and it worked right away.

**Comment by slinso** [comment]
I am trying to update the firmware for quite some time. First I tried a 64GB sdcard, but that did not work. Even with smaller partitions it was not working.
Then I switched to a 4GB sdcard and I was hoping for quick success after reading this discussion, but I am still not able to update the firmware.

Is there any way to debug the problem. I do not see anything regarding sdcard or flashing in the klippy.log

**Comment by JulianGroeli** [comment]
Mine worked with a 16GB card. Did you check the format is FAT32 and an
allocation size of 4096? Only one partition on the card?

Did you set the correct settings to create the .bin file?

Sorry, I don't know how to debug further.

Julian Groeli
